pkg/autoscaler/statforwarder/processor.go

```diff
@@ -75,30 +75,20 @@ type remoteProcessor struct {
 	bkt string
 	// holder is the HolderIdentity of a Lease for a bucket.
 	holder   string
-	svcDNS   string
+	addrs    []string
```

Comment:
Note that this is service and direct pod IP addresses? And they have to be ordered (pod, svc), since we prefer direct pod connections.

Reply:
I will add a comment about the relevance of ordering here, but the use of that ordering for pod/svc is implementation details of the consumer vs. a part of what this itself does.